src/components/views/rooms/RoomPreviewBar.tsx

```diff
@@ -524,9 +524,9 @@ export default class RoomPreviewBar extends React.Component<IProps, IState> {
                 }
 
                 const myUserId = MatrixClientPeg.get().getUserId();
-                const memberEventContent = this.props.room.currentState.getMember(myUserId).events.member.getContent();
+                const memberEventContent = this.props.room?.currentState.getMember(myUserId).events.member.getContent();
```

Comment:
From the interface `getMember`/`member` event can be `null`. Maybe just to be sure it should better be:

```suggestion
                const memberEventContent = this.props.room?.currentState.getMember(myUserId)?.events.member?.getContent();
```